student.rb

```diff
@@ -1,8 +1,13 @@
 require './person'
+require './classroom'
+
 class Student < Person
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+  attr_accessor :classroom
+
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
```

Comment:
- You have not properly designed a custom setter for the classroom attribute. Please fix this 🙏. You might find this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) useful in giving a better understanding 👍

Reply:
@mckent05 can you show me where it says create a custom setter for classroom???